apps/tailwind-components/components/content/ContentBlock.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <section
+    class="bg-white py-18 lg:px-12.5 px-5 text-gray-900 xl:rounded-3px last:rounded-b-50px last:mb-18 shadow-primary xl:border-b-0 border-b-[1px] overflow-hidden"
+  >
+    <h2 class="mb-5 uppercase text-heading-4xl font-display" v-if="title">
```

Comment:
I'm assuming that this component isn't used as a subsection. If see, we would need to make the heading element more flexible.